### ISSUE TITLE
When Inline, it does not wrap around the text

### ISSUE THREAD

**Issue by demiro** [bae-sh/tiptap-extension-resize-image#7]
When setting the image as inline, I would like to be able to wrap text around it (if I drag the image inside of a paragraph)...

like I found in this example
[https://codesandbox.io/p/sandbox/wrapping-text-around-an-image-using-html-2xj9yu?file=%2Findex.html](https://codesandbox.io/p/sandbox/wrapping-text-around-an-image-using-html-2xj9yu?file=%2Findex.html)

or like CKEditor does in basic example
[https://ckeditor.com/ckeditor-5/demo/editor-types/](https://ckeditor.com/ckeditor-5/demo/editor-types/)

How would I go about it? This is one of the bases of document formatting and I am quite unsure why tiptap does not provide this out of the box

**Comment by Im-younique** [comment]
Hello, @demiro ,

I am aware of the issue, but I don't think Tiptab currently supports that feature.

I know that it wasn't long ago that creating images inline was added as an option.

We will review this feature positively and try to add it.

It may take some time because I think I could make a big error if I touch it carelessly, but I will find a solution.

Thanks for leaving it as an issue! I'll leave a comment again with good news.

PR is always welcome. 🤗

**Comment by demiro** [comment]
@Im-younique Yeah, I thought as much. Weird that TipTap nor ProseMirror do not offer any support for this out of the box. This kind of functionality has been around in wysiwyg editors for ages. Think I will have to tackle this asap, as we are rolling the updates to the production and lacking this functionality is not really acceptable in our setting. If I do, will def. consider a PR if you don't bit me to it!

Thanks again for the hard work, I do appreciate it

**Comment by jedgrant** [comment]
FYI, this is just a CSS issue. If when you align left, you set the CSS to "float: left" or "float: right" the text will "wrap" around it.

**Comment by jedgrant** [comment]
@demiro - I added a PR. Not sure how good the code quality is, but it lets the text wrap around it when using inline = true. Works well enough for my needs.
